csharp/src/AddressBook/SampleUsage.cs

```diff
@@ -57,7 +57,11 @@ private static void Main()
             Person copy = Person.Parser.ParseFrom(bytes);
 
             // A more streamlined approach might look like this:
-            bytes = copy.ToByteArray();
+            AddressBook book = new AddressBook
+            {
+                People = { copy }
+            };
+            bytes = book.ToByteArray();

```

Comment:
please also remove the comment "// A more streamlined approach might look like this:" that doesn't make much sense anymore